app/Http/Controllers/UserController.php

```diff
@@ -19,13 +22,35 @@ public function __construct(
     /**
      * Display a listing of the resource.
      */
-    public function index()
+    public function index(Request $request)
     {
-        $res = $this->api
-            ->addParam(['order' => 'desc'])
-            ->sendRequest('users');
 
-        return $res;
+        $userSetting = \auth('sanctum')->user()->UserSetting;
+
+        if ($userSetting['requests_sent'] < $userSetting['max_requests']){
+
+            $response = $this->api
+                ->addParam(['order' => 'desc'])
+                ->sendRequest('users');
+
+            UserSettings::find($userSetting['id'])->update([
+                'requests_sent' => $userSetting['requests_sent']+1
```

Comment:
[Increment](https://laravel.com/docs/10.x/queries#increment-and-decrement)

$user = UserSettings::find($rameID);
$user->increment('column');

ან ჩააბი :D ლითერალი სილამაზეა უფრო ვიდრე სხვა რამ

Reply:
![image](https://github.com/AleksandreGaspariani/stackexchange_api/assets/87417497/9054fb62-25ff-4f2a-801d-575a31677587)

Reply:
@AleksandreGaspariani auth('sanctum')->user()->userSetting()->increment('request_sent'); eseti versia ar imushavebs ? relationships rom mimarto rogorc metods

Reply:
seems like it worked properly 🥲 thanks for advice sir  @Cosmonaut07

Reply:
Auth ფასადი არსებობს auth('sanctum') რომ არ წერო ყველა ჯერზე, 
Auth::user() გიბრუნებს ავტორიზებულ იუზერს.

user()->userSettings()
ორჯერ ვამბობთ იუზერებს და ცხრილს ერქვას userSettings -უბრალოდ 
User მოდელში რელაციას settings რომ დაარქვა user()->settings იქნება which reads a lot nicer